GitHub-Org-Management-Policy.md

```diff
@@ -48,22 +40,6 @@ Any repository created under the Node.js GitHub Organization is considered to be
 a project under the ownership of the Node.js Foundation, and thereby subject
 to the Intellectual Property and Governance policies of the Foundation.
 
-No repository may be deleted, transferred in to, or transferred out of, the
-Node.js Foundation GitHub Organization without a simple majority of both
-the TSC and CommComm in *favor* of the action. In certain cases, Node.js
-Foundation Board of Directors approval may also be required.
```

Comment:
I think this paragraph is still needed, without the CommComm reference.

Reply:
Reinstated. Thanks.